gcp-ts-oidc-provider-pulumi-cloud/index.ts

```diff
@@ -43,14 +44,19 @@ const oidcProvider = new gcp.iam.WorkloadIdentityPoolProvider(`identity-pool-pro
   workloadIdentityPoolId: identityPool.workloadIdentityPoolId,
   workloadIdentityPoolProviderId: `pulumi-cloud-${pulumi.getOrganization()}-oidc`,
   oidc: {
-    issuerUri: "https://api.pulumi.com/oidc",
+    issuerUri: issuer,
     allowedAudiences: [`gcp:${pulumi.getOrganization()}`],
   },
   attributeMapping: {
     "google.subject": "assertion.sub",
   },
 });
 
+const enableIamCredsApi = new gcp.projects.Service("enableIamCredentialsApi", {
+    service: "iamcredentials.googleapis.com",
+    project: gcpProjectName,
+});
```

Comment:
Suggest adding `retainOnDelete` as a courtesy since most people don't want to disable APIs once they are enabled.